src/js/components/TaskView.js

```diff
@@ -219,11 +241,7 @@ class TaskView extends mixin(SaveStateMixin, StoreMixin) {
       });
     });
 
-    if (searchString !== '') {
-      tasks = StringUtil.filterByString(tasks, 'id', searchString);
-    }
-
-    tasks = this.filterByCurrentStatus(tasks);
+    tasks = this.getFilteredTasks(filterByStatus, tasks, searchString);
```

Comment:
Ohh I had those in there when I thought we needed `getFilteredTasks` in `componentWillReceiveProps` and wanted to filter with nextProps, fixed!